movie/src/components/PopupCard/DotPopup/DotPopup.jsx

```diff
@@ -0,0 +1,21 @@
+import { Grid, Typography } from "@mui/material";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
+import "./DotPopup.css";
+const DotPopup = ({ handleClose }) => {
+  return (
+    <Grid className="cardPopup" onClick={handleClose} fullWidth>
+      <Typography>Want to rate or add this item to a list?</Typography>
+      <Grid className="btn">
```

Comment:
this can be quite confusing. its a `Grid` component but why does it has a className of `btn`